app/models/user.rb

```diff
@@ -2,6 +2,8 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  has_many :counselors
```

Comment:
Ahhhh shit shit yes you're right one user is only one counselor.
Apologies! Hang on.